templates/taxi/car_detail.html

```diff
@@ -1,22 +1,61 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
-  </h1>
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  {% if messages %}
+    <div>
+      {% for message in messages %}
+        <div class="alert {% if message.tags %}alert-{{ message.tags }}{% endif %}">
+          {{ message }}
+        </div>
+      {% endfor %}
+    </div>
+  {% endif %}
+  <h1>Model: {{ car.model }}</h1>
+  <p>Manufacturer: {{ car.manufacturer.name }} ({{ car.manufacturer.country }})</p>
+
+  <div class="d-flex justify-content-between align-items-center mb-3">
+    <h1 class="mb-0">Drivers</h1>
+
+    {% if user in car.drivers.all %}
```

Comment:
The condition `user in car.drivers.all` should be `user in car.drivers.all()` to properly call the `all()` method and retrieve the queryset of drivers.